holoviews/plotting/widgets/jsslider.css

```diff
@@ -1,71 +1,89 @@
 div.hololayout {
-    display: flex;
```

Comment:
Could you add some comments to mark which bits of CSS map to the sliders in jupyterlab and which map to the classic notebook?